src/EventHub/EventHub.Autorest/docs/Approve-AzEventHubPrivateEndpointConnection.md

```diff
@@ -215,31 +215,9 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## OUTPUTS
 
-### Microsoft.Azure.PowerShell.Cmdlets.EventHub.Models.Api202301Preview.IPrivateEndpointConnection
+### Microsoft.Azure.PowerShell.Cmdlets.EventHub.Models.IPrivateEndpointConnection
 
 ## NOTES
 
-ALIASES
```

Comment:
Also I am using the Autorest V4 to generate the PowerShell Command.
What can I see is the Notes in the docs are missing.
Why is so?

Reply:
@schaudhari6254888 
Yes, Notes have been removed in v4, since in online docs, users may refer to C# reference docs for detailed definitions of the class.
Regarding the link issues, these two links are from the swaggers. As @wyunchi-ms mentioned they are actually invalid links. Please either fix them or remove them from swaggers.
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2023-01-01-preview/namespaces-preview.json#L399

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2023-01-01-preview/namespaces-preview.json#L447